addons/binding/org.openhab.binding.feed/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="feed" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- DEFINITIONS of terms used in the binding:
+	    Feed is a XML document used for providing users with frequently updated content. It is located at URL. The most popular feed formats are RSS and Atom. 
+		Entry is a single element in the Feed, that contains reference (link), short description and other information like images, 
+		comments and etc. Entry in this binding is abstraction for RSS item element and Atom entry element. -->
+
+	<!-- Feed Thing Type -->
+	<thing-type id="feed">
+		<label>Feed</label>
+		<description>Provides the content of the Feed in XML format.</description>
+
+		<channels>
+			<channel id="feed-channel" typeId="feed-channel" />
+		</channels>
+
+		<config-description>
+			<parameter name="URL" type="text" required="true">
+				<label>Feed URL</label>
+				<description>Feed URL used to localize the Feed.</description>
+			</parameter>
+
+			<!--After the refresh time interval expires, the bindings checks for updates in the Feed, and if the information is not 
+				up to date, updates the feed content stored in the channel -->
+
+			<parameter name="refresh" type="integer">
+				<label>Refresh time interval</label>
+				<description>Refresh time interval in minutes.</description>
+				<default>20</default>
+			</parameter>
+
+			<!-- Feeds are converted to one of the following formats. Comparison between the two most popular formats - RSS 2.0 and 
```

Comment:
Hm, I think I am somehow missing the whole intention of this binding - it converts one format into another? What use is that for the user? If this is what it is doing, I would rather expect it to be a transformation service.

As a user, I would expect from such a binding to get the content (or even only the headline) of the latest post, so that I can display this in my UI or send a notification about it. Having XML within my StringItem is not really expected.